src/components/explorer/builder/QueryPickerContainer.js

```diff
@@ -236,6 +275,40 @@ class QueryPickerContainer extends React.Component {
               >
                 <FormattedMessage {...localMessages.addQuery} />
               </a>
+              <IconButton
+                aria-label="more"
+                aria-controls="long-menu"
+                aria-haspopup="true"
+                onClick={(evt) => this.setState({ quickCompareMenuAnchorEl: evt.currentTarget, quickCompareMenuOpen: true}) }
+              >
+                <MoreVertIcon />
+              </IconButton>
+              {this.state.quickCompareMenuOpen && (
+                <>
+                  <Menu
+                    id="simple-menu"
```

Comment:
There's nothing technically wrong with this id name, but it might prevent unique ID conflicts if changed to something more specific, like "comparison-menu"